dnssec.tf

```
@@ -1,17 +1,15 @@
-data "aws_caller_identity" "kms" {
-  provider = aws.kms
-}
+data "aws_caller_identity" "kms" {}
```

```suggestion
data "aws_caller_identity" "current" {}
```

Can we change this to the default we use, whether it be `default` or `current`, if no longer using a region specific resource?